dev/org.eclipse.codewind.core/src/org/eclipse/codewind/core/internal/InstallUtil.java

```diff
@@ -73,6 +77,48 @@ public static ProcessResult stopCodewind(IProgressMonitor monitor) throws IOExce
 		}
 	}
 	
+	public static ProcessResult installCodewind(IProgressMonitor monitor) throws IOException, TimeoutException {
+		SubMonitor mon = SubMonitor.convert(monitor, "Installing Codewind", 100);
```

Comment:
This string should be in a message file.  Can you also put the string in startCodewind in the message file as well?